src/openassetio-core/tests/VersionTest.cpp

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 The Foundry Visionmongers Ltd
+#include <catch2/catch.hpp>
+
+#include <openassetio/Version.hpp>
+
+SCENARIO("OpenassetIO exposes version information") {
+  REQUIRE(openassetio::majorVersion() == OPENASSETIO_VERSION_MAJOR);
```

Comment:
[consider] `REQUIRE` will abort the test and not check the other assertions.  I think `CHECK` would be better here.

Reply:
I don't need to re-review if you change these.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1247/commits/b93c4319138a58aad45d804c7ebe4ab373b20102